frontend/app/src/components/asset-manager/MergeDialog.vue

```diff
@@ -8,9 +8,9 @@ type Errors = Partial<
   Record<'targetIdentifier' | 'sourceIdentifier', string[]>
 >;
 
-const props = defineProps<{ value: boolean }>();
+const props = defineProps<{ modelValue: boolean }>();
 
-const emit = defineEmits<{ (e: 'input', value: boolean): void }>();
+const emit = defineEmits<{ (e: 'update:model-value', value: boolean): void }>();
 
 const display = useSimpleVModel(props, emit);
```

Comment:
defineModel

Reply:
I did some of them but we can do the rest slowly